app.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "workshop-portal",
   "scripts": {
+    "postdeploy": "bundle exec rake db:schema:load db:seed"
```

Comment:
Depending on when this setting applies we might want to add `db:populate_sample_data`